modules/simple_forms_api/app/form_mappings/vba_21_4140.json.erb

```diff
@@ -1,9 +1,14 @@
 {
-  "F[0].Page_1[0].Station_Address[0]": "<%= form.data.dig('Station_Address') %>",
 
   <%# Page 1 %>
 
-  <%# Section 1: Veteran's Identification Information %>
+  <%# Station Address %>
+  "F[0].Page_1[0].Station_Address[0]": "<%= %>",
+
+  <%# Date Mailed %>
+  "F[0].Page_1[0].Date_Mailed[0]": "<%= %>",
```

Comment:
These are not applicable to digital form submissions and hence intentionally left blank.